### PR TITLE
Adding support for IBM i PASE installs

### PR DESCRIPTION
**Pull request by ab-m**
Pull Request in response to issue #537

### CODE REVIEW

File: src/appmetrics.cpp [hunk 1]
```diff
@@ -285,7 +285,7 @@ static void* getMonitorApiFunction(std::string pluginPath, std::string functionN
     std::string libname = "hcapiplugin.dll";
 #elif defined(__MACH__) || defined(__APPLE__)
     std::string libname = "libhcapiplugin.dylib";
-#elif defined (__AIX__) || defined(AIX)
```
Comment (sjanuary) [inline src/appmetrics.cpp]: @ab-m - what's the purpose of this change?

Comment (ab-m) [reply src/appmetrics.cpp]: @sjanuary `_AIX` is predefined on both AIX & IBM i PASE systems whereas `AIX` is not predefined on. In this instance both AIX & PASE would follow the same code path so `_AIX` was used.

Alternatively another `|| defined(__PASE__)` could have been appended to the line instead but the first approach using `_AIX` seemed cleaner.

Comment (sjanuary) [reply src/appmetrics.cpp]: that's fine, I just wanted to understand the reason for it

**Comment by sjanuary** [comment]
@ab-m I have tagged a new omr-agentcore release so you can now update the submodule in your branch to point to that tag.  Also if you could add something to the README about iPASE that would be great, i.e. what works and what doesn't work.
